### PR TITLE
TD-1751-Acceptable use policy, link added.

### PR DESCRIPTION
**Pull request by Auldrin-Possa**
**JIRA link**
https://hee-tis.atlassian.net/browse/TD-1751

**Description**
Feature 'Acceptable use policy' added.

**Screenshots**
image

image

image

-----
**Developer checks**

I have:
- [x] Run the formatter and made sure there are no IDE errors
- [ ] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [x] Manually tested my work with and without JavaScript
- [x] Tested any Views or partials created or changed with [Wave Chrome plugin]
- [ ] Updated/added documentation in [Confluence]
- [ ] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken
- [x] Scanned over my pull request in GitHub and addressed any warnings from the GitHub Build and Test checks.

### CODE REVIEW

File: DigitalLearningSolutions.Web/Views/Shared/_DlsFooter.cshtml [hunk 1]
```diff
@@ -9,6 +9,9 @@
                 <li class="nhsuk-footer__list-item">
                     <a class="nhsuk-footer__list-item-link" target="_blank" asp-controller="LearningSolutions" asp-action="Terms">Terms of use <span class="nhsuk-u-visually-hidden">(opens new browser window)</span></a>
                 </li>
+                <li class="nhsuk-footer__list-item">
+                    <a class="nhsuk-footer__list-item-link" target="_blank" asp-controller="LearningSolutions" asp-action="AcceptableUsePolicy">Acceptable Use Policy <span class="nhsuk-u-visually-hidden">(opens new browser window)</span></a>
```
Comment (rshrirohit) [inline DigitalLearningSolutions.Web/Views/Shared/_DlsFooter.cshtml:13]: Rename "Acceptable Use Policy" to "Acceptable use policy" in order to be consistent with casing used in other lables in the footer

**Review by rshrirohit — CHANGES_REQUESTED** [review]

File: DigitalLearningSolutions.Data.Migrations/202307101239_AddAcceptableUsePolicyRecord.cs [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+                                            <li>immediate, temporary, or permanent removal of any Contribution uploaded by you to the Platform;</li>
+                                            <li>issue of a warning to you;</li>
+                                            <li>legal proceedings against you for reimbursement of all costs on an indemnity basis (including, but not limited to, reasonable administrative and legal costs) resulting from the breach, and/or further legal action against you;</li>
+                                            <li>disclosure of such information to law enforcement authorities as we reasonably feel is necessary or as required by law; and/or</li>
+                                            <li>any other action we reasonably deem appropriate.</li>
+                                        </ol>
+                                    </li>
+                                </ol>";
+
+            Execute.Sql(@"INSERT INTO [dbo].[Config] ([ConfigName], [ConfigText], [IsHtml])
```
Comment (rshrirohit) [inline DigitalLearningSolutions.Data.Migrations/202307101239_AddAcceptableUsePolicyRecord.cs:213]: Add updated date here to current date.

Comment (Auldrin-Possa) [reply DigitalLearningSolutions.Data.Migrations/202307101239_AddAcceptableUsePolicyRecord.cs]: For insert operation on Config table, default values for CreateDate and UpdateDate are current dates. Added both dates explicitly. 
Please review.

File: DigitalLearningSolutions.Web/Views/Shared/_DlsFooter.cshtml (hunk 1 above)
Comment (Auldrin-Possa) [reply DigitalLearningSolutions.Web/Views/Shared/_DlsFooter.cshtml]: Link text updated. Please review.

**Comment by Auldrin-Possa** [comment]
> 1. The alignment needs to be corrected -
> 2. Everything appears to be in bold -
> 3. Color for hyperlinks is too dark, please check other pages and make it same -
>    image
> 
> Please correct the numbering in the page - image

Note: Wanted to check if all above issues are present even after running 'yarn dev' command.

**Pull request merged**